frontend/simple/views/components/Lists/index.js

```diff
@@ -0,0 +1,2 @@
+export { default as List } from './List.vue'
+export { default as ListItem } from './ListItem.vue'
```

Comment:
Would it be possible to avoid these `index.js` files?

Reply:
I mean, it feels like there might be a cleaner way of doing this... However, I see that you at least do fully specify the filename and extension when importing, so it's not too bad... just wondering if there's some other way. I'll keep thinking on it as I review the PR in more detail.

Reply:
This is the cleaner way I know to export multiple named modules. Maybe @hubudibu knows something better?

Reply:
I also prefer these index.js files, it helps keep the imports where these components are used shorter. it makes it possible to do
```js
import { List, ListItem } from '../../components/Lists'
```
instead of having to do longer imports for each line like
```js
import List from '../../components/Lists/List.vue'
import ListItem from '../../components/Lists/ListItem.vue'
```
it also makes it more explicit at the imports where we use these that they go together.

Reply:
Yes but per the recent discussions all extensions must be specified, so we can't do `import { List, ListItem } from '../../components/Lists'`, it must be `import { List, ListItem } from '../../components/Lists/index.js'` for future compatibility and also clarity. But it's fine to do it that way. 👍

Reply:
Kidoki, I'll add the `index.js`.

Reply:
Added